pom.xml

```diff
@@ -102,7 +104,6 @@
       <url>http://repo.jenkins-ci.org/public/</url>
     </repository>
   </repositories>
-
```

Comment:
:-1:

Reply:
It this :-1: for just this line or the whole file?